packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -21,6 +21,16 @@ final authRef = create(Auth.new);
 // The [Auth] instance available in the current zone.
 Auth get auth => read(authRef);
 
+/// Matches 'https://accounts.google.com' exactly.
+final googleJwtIssuerRegexp = RegExp(r'https:\/\/accounts\.google\.com');
+
+/// Matches lines like
+/// https://login.microsoftonline.com/{tenant-id}/v2.0/.well-known/openid-configuration
+/// Captures the tenant ID as a group.
+final microsoftJwtIssuerRegexp = RegExp(
+  r'https:\/\/login\.microsoftonline\.com\/([\w-]+)\/v2\.0',
```

Comment:
I don't think they would. I've updated to no longer use regex and to do an exact match for google (because the issuer is always `https://accounts.google.com`) and to use a prefix match for microsoft (whose issuers are of the form `https://login.microsoftonline.com/{tenant-id}/v2.0`.